pkg/controllers/workgenerator/override_test.go

```diff
@@ -2114,6 +2114,37 @@ func TestApplyJSONPatchOverride(t *testing.T) {
 				},
 			},
 		},
+		{
+			name: "reset the labels",
+			deployment: appsv1.Deployment{
+				TypeMeta: deploymentType,
+				ObjectMeta: metav1.ObjectMeta{
+					Name:      "deployment-name",
+					Namespace: "deployment-namespace",
+					Labels: map[string]string{
+						"app": "nginx-1",
+						"key": "value",
+					},
+				},
+			},
+			overrides: []placementv1alpha1.JSONPatchOverride{
+				{
+					Operator: placementv1alpha1.JSONPatchOverrideOpAdd,
```

Comment:
Just curious, will it have similar effect to use `JSONPatchOverrideOpReplace`?

Reply:
it seems this is more of a "replace" semantic, the add op is kinda weird

Reply:
added another test and comment.

The main diff between "Replace" and "Add" is that add works even the path is missing.